### PR TITLE
adding instructions for PS

### PR DESCRIPTION
**Pull request by wduraes**
Added instructions to run PowerShell and moved the ESP8266 board setup to pre-requisites.

### CODE REVIEW

File: sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md [hunk 1]
```diff
@@ -29,6 +29,12 @@ _The following was run on Windows 10 and Ubuntu Desktop 20.04 environments, with
     NOTE: Device keys are used to automatically generate a SAS token for authentication, which is only valid for one hour.
 
 - Have the latest [Arduino IDE](https://www.arduino.cc/en/Main/Software) installed.
+
+- Install the Esp8266 board support on Arduino IDE.
```
Comment (danewalton) [inline sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md:33]: nit:
unify capitalization or non-capitalization (IE `Esp8266` or `ESP8266`)

File: sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md [hunk 2]
```diff
@@ -56,11 +62,24 @@ _The following was run on Windows 10 and Ubuntu Desktop 20.04 environments, with
...
+    PS C:\> .\New-ArduinoZipLibrary.ps1
+    ```
+
+    Note that in several cases, script execution is restricted by default for security reasons. If you can't run the script above, then run PowerShell as Administrator:
+
+    ```powershell
+    Set-ExecutionPolicy Unrestricted
+    ```
+    
+    In this case, don't forget to move the security settings back once you complete the setup:
```
Comment (danewalton) [inline sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md:79]:
```suggestion
    In this case, don't forget to move the security settings back once you complete the setup if you wish:
```

Some people might like to keep it on to be able to run them in the future?

**Review by danewalton — APPROVED** [review]

File: sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md (hunk 1 above)
Comment (momuno) [inline sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md:33]: I would keep the same sentence structure as other bullet points in section:
```suggestion
- Have the [ESP8266 board support](https://github.com/esp8266/Arduino#installing-with-boards-manager) installed on Arduino IDE. ESP8266 boards are not natively supported by Arduino IDE, so you need to add them manually.
```

File: sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md [hunk 3]
```diff
@@ -29,6 +29,12 @@ _The following was run on Windows 10 and Ubuntu Desktop 20.04 environments, with
     NOTE: Device keys are used to automatically generate a SAS token for authentication, which is only valid for one hour.
 
 - Have the latest [Arduino IDE](https://www.arduino.cc/en/Main/Software) installed.
+
```
Comment (momuno) [inline sdk/samples/iot/docs/how_to_iot_hub_esp8266_nodemcu.md:32]: Extra new line not necessary.

**Review by momuno — APPROVED** [review]

**Pull request merged**